packages/maskbook/src/web3/types.ts

```diff
@@ -8,6 +10,10 @@ export enum CurrencyType {
     USD = 'usd',
 }
 
+export interface MetaMaskInpageProvider extends ReturnType<typeof createMetaMaskProvider> {
```

Comment:
I think `src/env.d.ts` is a better place for this.

Reply:
I think this is web3 related type.